### PR TITLE
Add SFX to some ui

### PR DESCRIPTION
**Pull request by sonohoshi**
### Description

SSIA

#### Menu
    - [sfx_click] every backbutton
#### Combination
    - [sfx_click] Combination main - Craft & Upgrade
#### Upgrade
    - [sfx_click] Grade filter
    - [sfx_click] Element filter
    - [sfx_chainmail] Enter Upgrade item
    - [sfx_chainmail] Enter Material item
#### Crafting UI
    - [sfx_click] OK button
#### Ranking
    - [sfx_click] Menu & submenu
#### Shop buy
    - [sfx_click] Menu & submenu
#### Character
    - [sfx_click] equipmentslot
    - [sfx_click] costumeslot

### How to test
- Click them

### Related Links

[[기타] sfx 추가](ㅁㅇㄴㄹ)

### CODE REVIEW

File: nekoyume/Assets/_Scripts/UI/ChatPopup.cs [hunk 1]
```diff
@@ -50,5 +51,11 @@ public override void Show(bool ignoreShowAnimation = false)
         {
             base.Show(true);
         }
+
+        public override void Close(bool ignoreCloseAnimation = false)
+        {
+            base.Close(ignoreCloseAnimation);
+            AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/ChatPopup.cs:58]: It is recommended to place this line in the callback lambda method of `confirm.onClick`.

File: nekoyume/Assets/_Scripts/UI/CombinationSlots.cs [hunk 2]
```diff
@@ -64,6 +75,17 @@ public bool TryGetEmptyCombinationSlot(out int slotIndex)
             return false;
         }
 
+        public override void Close(bool ignoreCloseAnimation = false)
+        {
+            if (blur && blur.isActiveAndEnabled)
+            {
+                blur.Close();
+                AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/CombinationSlots.cs:83]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Mail.cs [hunk 3]
```diff
@@ -115,9 +116,10 @@ public override void Show(bool ignoreShowAnimation = false)
 
         public override void Close(bool ignoreCloseAnimation = false)
         {
-            if (blur)
+            if (blur && blur.isActiveAndEnabled)
             {
                 blur.Close();
+                AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Mail.cs:122]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs [hunk 4]
```diff
@@ -96,6 +97,7 @@ protected void Awake()
                 {
                     _grade = filter;
                     SortedData(_grade, _elemental);
+                    AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs:100]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs [hunk 5]
```diff
@@ -116,8 +118,14 @@ protected void Awake()
                 {
                     _elemental = filter;
                     SortedData(_grade, _elemental);
+                    AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs:121]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs [hunk 6]
```diff
@@ -350,6 +351,7 @@ private void OnClick(BaseEventData eventData)
             {
                 case 1:
                     _onClick?.Invoke(this);
+                    AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs:354]: No need play click sounds when double clicked?

File: nekoyume/Assets/_Scripts/UI/Rank.cs [hunk 7]
```diff
@@ -156,10 +164,26 @@ public override void Initialize()
...
             base.Show(ignoreShowAnimation);
             UpdateCategory(RankCategory.Ability, true);
         }
 
+        public override void Close(bool ignoreCloseAnimation = false)
+        {
+            if (blur && blur.isActiveAndEnabled)
+            {
+                blur.Close();
+                AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Rank.cs:181]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Settings.cs [hunk 8]
```diff
@@ -260,9 +261,10 @@ public void RedeemCode()
 
         public override void Close(bool ignoreCloseAnimation = false)
         {
-            if (blur)
+            if (blur && blur.isActiveAndEnabled)
             {
                 blur.Close();
+                AudioController.PlayClick();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Settings.cs:267]: It is recommended to place this line in the input event method.

File: nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs (hunk 6 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs:354]: Yes. Double-click executes equip/unequip with chainmail sfx.

File: nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs (hunk 4 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs:100]: This line already placed in `onValueChanged` event method.

File: nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs (hunk 5 above)
Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/EquipmentInventory.cs:121]: This line already placed in onValueChanged event method.

**Review by Unengine — APPROVED** [review]

**Review by Namyujeong — APPROVED** [review]
Some backbutton are doesn't apply the sfx, but it would be better to create another PR.

**Pull request merged**